guides/source/testing.md

```diff
@@ -146,18 +146,28 @@ Let us run this newly added test (where `6` is the number of line where the test
 
 ```bash
 $ bin/rails test test/models/article_test.rb:6
+Run options: --seed 44656
+
+# Running:
+
 F
 
-Finished tests in 0.044632s, 22.4054 tests/s, 22.4054 assertions/s.
+Failure:
+ArticleTest#test_should_not_save_article_without_title [/path/to/blog/test/models/article_test.rb:6]:
+Expected true to be nil or false
 
-  1) Failure:
-test_should_not_save_article_without_title(ArticleTest) [test/models/article_test.rb:6]:
-Failed assertion, no message given.
 
-1 tests, 1 assertions, 1 failures, 0 errors, 0 skips
+bin/rails test test/models/article_test.rb:6
+
+
+
```

Comment:
Hmm, are the extra lines on every run? They look uneven when compared to the output from "Error" below.